main/zoe/guide/contracts/auto-swap.md

```diff
@@ -28,21 +28,24 @@ methods in the API available to the user:
 3. getPrice
```

Comment:
Let's update the section above this which explains how to make an instance. Making an instance returns an invite. We could probably leave that section out if we want to, as well. 

Let's update this numbered section talking about the API, too. We should split it in the invite API (addLiquidity, removeLiquidity, swap) and the publicAPI: getPrice,  getLiquidityAssay, getPoolUnits, and makeInvite.

Reply:
I think we should leave the initialization part because the next section references it.

Is this what you mean?

```
The moola<->simolean autoswap invite that we just created has a number of
methods available:

#### Invite API:
1. addLiquidity
2. removeLiquidity
3. swap

#### Public API:
1. getPrice
2. getLiquidityAssay
3. getPoolUnits
4. makeInvite
```

Reply:
Yeah, this is perfect!